programming-extensions/programming-extension-dataspaces/src/main/java/org/objectweb/proactive/extensions/dataspaces/vfs/VFSNodeScratchSpaceImpl.java

```diff
@@ -293,14 +302,27 @@ public synchronized void close() throws IllegalStateException {
             this.fileSystemManager.close();
         }
         logger.debug("Closed node scratch space");
+
+        closed = true;
```

Comment:
They behave like they were behaving before. The `close` field was introduced to make the  `close` method idempotent since a change with the new version of VFS makes successive `close` operations fail if it is not the case. Changing the semantic of other methods that pass tests will require more time and is too risky to me.